01_Data/src/layers/sequelize/model/Location/SetNetworkProfile.ts

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2023 S44, LLC
+// Copyright Contributors to the CitrineOS Project
+//
+// SPDX-License-Identifier: Apache 2.0
+
+import { OCPPInterfaceEnumType, OCPPTransportEnumType, OCPPVersionEnumType, APNType, VPNType, NetworkConnectionProfileType } from '@citrineos/base';
+import { Column, DataType, ForeignKey, Index, Model, Table } from 'sequelize-typescript';
+import { ChargingStation } from './ChargingStation';
+import { ServerNetworkProfile } from './ServerNetworkProfile';
+
+/**
+ * The CallMessage model can be extended with new optional fields,
+ * e.g. chargingProfileId, for other correlationId related lookups.
+ */
+@Table
+export class SetNetworkProfile extends Model {
+    static readonly MODEL_NAME: string = 'CallMessage';
```

Comment:
How about implements `NetworkConnectionProfileType` for this class and add it in `namespace.ts`? Then use it as the model name here? Or we just replace with 'SetNetworkProfile' for model name.